library/src/main/java/com/getstream/sdk/chat/ChatSocketListener.kt

```diff
@@ -141,7 +142,6 @@ internal class ChatSocketListener(
             is ConnectingEvent,
             is DisconnectedEvent,
             is ErrorEvent,
-            is ChannelUpdatedByUserEvent,
```

Comment:
Why this event is not handled anymore?
Does it compile?

Reply:
It was a duplicate in the list of events, listed twice in the same branch. Removing it removes a warning.

Reply:
It seems that was added by two different PRs